server/sandbox_run.go

```diff
@@ -331,13 +331,19 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	cgroupParent := req.GetConfig().GetLinux().CgroupParent
 	if cgroupParent != "" {
 		if s.config.CgroupManager == oci.SystemdCgroupsManager {
-			cgPath, err := convertCgroupNameToSystemd(cgroupParent, false)
+			if !strings.HasSuffix(path.Base(cgroupParent), ".slice") {
```

Comment:
`if len(config.CgroupParent) <= 6 || !strings.HasSuffix(config.CgroupParent, ".slice")`

Reply:
Would it be at all useful to have different error messages for these conditions?  I think with this suggestion a small slice would trigger the below error saying we'd not received a slice when we in fact had.  Or maybe I've not had enough (any) tea yet this morning.

Reply:
the check I'm suggesting is to handler `cgroupParent = ".slice"` - docker handles that as well.